packages/aws-cdk/README.md

```diff
@@ -560,6 +561,126 @@ This feature currently has the following limitations:
+be within a generated subdirectory in your current working directory unless `--output-path` is specified.
+If a directory already exists with the same name as `--stack-name`, it will be replaced with the new application.
+All CDK supported languages are supported, language choice can be specified with `--language`.
+
+#### Generate a typescript application from a local template.json file
+
+```console
+$ # template.json is a valid cloudformation template in the local directory
+$ cdk migrate --stack-name MyAwesomeApplication --language typescript --from-path MyTemplate.json
+```
+
+This command will generate a new directory named `MyAwesomeApplication` within your current working directory, and  
+then initialize a new CDK application within that directory which has the same resource configuration
+as the provided template.json
+
+This results in a CDK application with the following structure, where the lib directory contains a stack definition
+with the same resource configuration as the provided template.json.
+
+```console
+├── README.md
+├── bin
+│   └── my_awesome_application.ts
+├── cdk.json
+├── jest.config.js
+├── lib
+│   └── my_awesome_application-stack.ts
+├── package.json
+├── tsconfig.json
+```
+
+#### Generate a python application from a deployed stack
+
+If you already have a CloudFormation stack deployed in your account and would like to manage it with CDK, you can use the 
+`--from-stack` option to generate the application. In this case the `--stack-name` must match the name of the deployed stack.
+
+```console
+$ # generate a python application from MyDeployedStack in your account
+$ cdk migrate --stack-name MyDeployedStack --language python --from-stack
+```
+
+This will generate a Python CDK application which will synthesize the same configuration of resources as the deployed stack.
+
+#### **CDK Migrate Limitations**
+
+- CDK Migrate does not currently support nested stacks, custom resources, or the `Fn::ForEach` intrinsic function.
+
+- CDK Migrate will only generate L1 constructs and does not currently support any higher level abstractions.
+
+- CDK Migrate successfully generating an application does *not* guarantee the application is immediately deployable.
+It simply generates a CDK application which will synthesize a template that has identical resource configurations 
+to the provided template. 
+
+- CDK Migrate does not interact with the CloudFormation service to verify the template 
+provided can deploy on its own. This means CDK Migrate will not verify that any resources in the provided 
+template are already managed in other CloudFormation templates, nor will it verify that the resources in the provided
+template are available in the desired regions, which may impact ADC or Opt-In regions. 
+
+- If the provided template has parameters without default values, those will need to be provided
+before deploying the generated application.
```

Comment:
These two feel like they should be sub-bullets of the previous bullet. Can you indent them that way?